boards/arm/mimxrt1050_evk/pinmux.c

```diff
@@ -109,6 +109,15 @@ static int mimxrt1050_evk_init(struct device *dev)
 	CLOCK_EnableClock(kCLOCK_Iomuxc);
 	CLOCK_EnableClock(kCLOCK_IomuxcSnvs);
 
+#if CONFIG_KSCAN_FT5336_INTERRUPT
```

Comment:
Can this be `#if DT_IRQ_HAS_IDX(DT_INST(0, kscan_ft5336), 0)`

Reply:
This is a gpio, not an interrupts property. Updated to `#if DT_NODE_HAS_PROP(DT_INST(0, focaltech_ft5336), int_gpios)` instead.